docs/osf-trivia-list/dasharo.md

```diff
@@ -21,3 +21,9 @@ it may consist of:
 * Fixes introducing Dasharo SMBIOS/DMI tables format according to [Dasharo
   Product Guidelines](../../dev-proc/smbios-rules)
 * Fixes for submodules building
+
+Those improvements are stored on `dasharo/common-base` branch in relevant
```

Comment:
Other names I was considering: `dasharo/dasharo`, `dasharo/base`

Reply:
Another proposal:

A set of patch files applicable on the recent stable coreboot tag (from which we should base all new ports, always a stable tag as a base for new port, Dasharo release). They would be stored on e.g. dasharo/dasharo repository. coreboot seems not to be a good place as it would introduce another perm branch.

These patches could conform to release cycles, i.e. a new release of patches would occur on a new coreboot upstream release quarterly.

Reply:
> Are you sure we should base on stable coreboot tag? I thought we had that discussion, and we decided that there is too much going on in coreboot to base on "stable" branch, which has no guarantees. IMO there is no logical justification of that statement instead of having upstream-managed known point in time. There is no other value in stable IMO.

I'm quite surprised to hear that, as my impression was that you always opted in for the tag, rather than some arbitrary commit.

Reply:
If we want to name it `Dasharo Common Base` then `dasharo/common-base` branch totally makes sense to me.

Reply:
@macpijan yes I did opted for for tag until I realized there is no way of changing mindset in coreboot leadership and community. Tags in coreboot are and will be just arbitrary point in time, so if we choose any other arbitrary point the value is the same, we just have smaller difference in code to take care what should mean easier upstream. Anything we need in Dasharo Common Base should be considered for upstream e.g. DMI settings could be just Kconfig options.

What we can think in future is having wide quality tests across Dasharo targets which can claim about quality of coreboot, if there will be any difference. Then based on test results for Dasharo targets we could judge if there is any place in the tree which is better than the other, but this is long story which means establishing quality control for huge project.